File: Pelikula.API/Model/Dojam/DojamUpsertRequest.cs
using System;

namespace Pelikula.API.Model.Dojam
{
    public partial class DojamUpsertRequest
    {
        public int Ocjena { get; set; }
        public string Tekst { get; set; }
        public DateTime Datum { get; set; }

        public int KorisnikId { get; set; }
        public int ProjekcijaId { get; set; }
    }
}

File: Pelikula.MOBILE/lib/model/dojam/dojam_upsert_request.dart
class DojamUpsertRequest {
  int? ocjena;
  String? tekst;

  int? korisnikId;
  int? projekcijaId;

  DojamUpsertRequest({
    this.ocjena,
    this.tekst,
    this.korisnikId,
    this.projekcijaId,
  });

  factory DojamUpsertRequest.fromJson(Map<String, dynamic> json) {
    return DojamUpsertRequest(
      ocjena: json['ocjena'] as int,
      tekst: json['tekst'] as String?,
      korisnikId: json['korisnikId'] as int,
      projekcijaId: json['projekcijaId'] as int,
    );
  }

  Map<String, dynamic> toJson() => {
        "ocjena": ocjena,
        "tekst": tekst,
        "korisnikId": korisnikId,
        "projekcijaId": projekcijaId,
      };
}
